en/ui-system/components/engine/ui-batch.md

```diff
@@ -1,5 +1,19 @@
-# UI Batch Rules
+# 2D Rendering Component Batching Rules
 
-The rules for UI batching are the same hash material and image and sampler for batching. The same hash means that even if the same material is used, if their hash values ​​are different, the batch will be interrupted. For example, two materials involve the modification of `Uniform`, resulting in failure to batch. Sprite and text are the same because their images are different. Of course, it can be done under strict control, but this is another story.
+## Batching Condition Description
 
-UI rendering data collection is based on the Hierarchy, so in the process of recursive Hierarchy, if you encounter the three components `UIMeshRenderer`, `Mask`, `Graphics` and the above situation, the batch will be interrupted. Therefore, we recommend that the images of different modules be batched separately to reduce DrawCall. However, if you want to batch a large amount of data, you need to strictly layout the nodes.
+The rules for batching 2D render components are that the layer of the node, the material used, the blendState and DepthStencilState of the render component are the same, and the map source and map sample are the same.
+We will explain each of these conditions below:
+- Layer of node: since layer is involved in rendering or not, different layers cannot be batched.
+- Material: the same material is a necessary requirement for batching, because we use the material instantiation mechanism, so the material will be instantiated after the user sets the uniform of the material, and the material after instantiation cannot be batching. After the uniform value is used, you can reassign the material resource to the component to make the component participate in the batch again (through the CustomMaterial interface).
```

Comment:
```suggestion
- Material: the same material is a necessary requirement for batching, because the material instantiation mechanism is used, the material will be instantiated after the user sets the uniform of the material, and the material after instantiation cannot be batching. After the uniform value is used, reassign the material resource to the component to make the component participate in the batch again (through the CustomMaterial interface).
```